src/ca/ubc/cs304/database/AddTablesAndData.sql

```diff
@@ -202,29 +202,29 @@ INSERT INTO TIMEPERIOD (fromDate, fromTime, toDate, toTime)
 VALUES ('2019-11-27', '12:05', '2019-01-20', '1:00');
 
 
-INSERT INTO Reservations (confNo, VTName, driversLicense, fromDate, fromTime, toDate, toTime)
+INSERT INTO Reservation (confNo, VTName, driversLicense, fromDate, fromTime, toDate, toTime)
```

Comment:
@taran1598  is this still applicable?

Reply:
I changed all that stuff on my branch.  Just make sure to not delete any additions in my branch otherwise a lot of clerk won't work